tsconfig.json

```diff
@@ -17,5 +18,5 @@
     "allowUmdGlobalAccess": true
   },
   "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules", "src/templates/**"]
```

Comment:
just exclude the example templates 7bb6052ea673928486900dcfa9b399ad9c6c3d49 so that the abstraction still checked by ts

Reply:
you forgot to delete the `src/`

Reply:
Fixed :Dead: